css homework#2

Danik2021 opened the pull request:
I did css (homework#2)

Danik2021 committed:
css homework#2

Danik2021 committed:
I did all css

!

Denys320 commented on image/normalize.css:
```diff
@@ -0,0 +1,349 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```
молодец!

Denys320 commented on index.html:
```diff
@@ -116,11 +112,11 @@ <h3 class="employee">Михайло Єрмаков</h3>
+        <a class="phone link" href="tel:+380961111111">+38 096 111 11 11</a>
       </div>
     </footer>
   </body>
```
script должен быть внутри body

xrustics replied:
![Screenshot_2024-02-18-00-05-01-995_com android chrome](https://github.com/Danik2021/goit-markup-hw-01/assets/59570719/659dff4d-5baa-4ec8-b8ee-22f3e8605270)

xrustics replied:
Поддерживаю, немного позже в программе будет эта тема чуть больше раскрыта, пока этой информации (скрин выше) будет достаточно для базового понимания почему так)

Danik2021 replied:
✔✔✔

Denys320 commented on index.html:
```diff
@@ -7,106 +7,102 @@
+      href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,100..900;1,100..900&display=swap"
       rel="stylesheet"
     />
     <link rel="stylesheet" href="style.css" />
```
А2. в проекте должна быть папка с файлами стилей

Danik2021 replied:
✔✔✔

Denys320 commented on index.html:
```diff
@@ -7,106 +7,102 @@
-          <h2>Чим ми займаємося?</h2>
-          <ul class="ul-img-work">
+          <h2 class="whatDo-title">Чим ми займаємося</h2>
+          <ul class="ul-img-work list">
```
я думаю название класса очень сложное, можно придумать что то попроще и чтобы было понятнее кому то другому, например 'work-name', у списка ul 'work-list'

xrustics replied:
@Danik2021 пересмотри пожалуйста комментарий в CSS файле по этим классам. Мне кажется тут можно сделать намного проще с одним классом)

Danik2021 replied:
✔✔✔

Denys320 commented on style.css:
```diff
@@ -0,0 +1,168 @@
+/* Body */
+body {
+  font-family: var(--main_font_family);
+  color: var(--main_text_color);
```
в боди можно добавить леттер спейс стандартный, а там дальше где нужен будет другой то просто его указать, вместо того чтобы он был везде указан и визуально нагружал код.
точно так же с фонт вейт и другими параметрами в будущем

Denys320 commented on style.css:
```diff
@@ -0,0 +1,168 @@
+  font-size: 14px;
+  line-height: 1.7;
+  letter-spacing: 0.03em;
+  color: #757575;
```
тут цвет текста является одним из основных, его так же можно добавить в body и везде указывать через var

Danik2021 replied:
✔✔✔

Denys320 commented on style.css:
```diff
@@ -0,0 +1,168 @@
+
+.span-logo {
+  font-family: inherit;
+  color: #2196f3;
```
так же нужно добавить через var, чтобы не путать никого и самому не запутаться в будущем

Danik2021 replied:
✔✔✔

xrustics commented on style.css:
```diff
@@ -0,0 +1,169 @@
+.main-title,
+.whatDo-title,
+.aboutUs-title,
+.ourTeam-title {
```
Я бы тут переделал. Зачем тебе так много классов, чтоб указать на одни и те же стили? Просто сделай один класс, например "section-title" и его можно свободно переиспользовать всюду)

Danik2021 replied:
✔✔✔

xrustics commented on style.css:
```diff
@@ -0,0 +1,169 @@
+  font-family: var(--main_font_family);
+}
+
+.cont-aboutUs-section p {
```
В CSS лучше придерживаться какого-то одного вида форматирования имени классов, например "cont-about-us-section". CamelCase тут ни к чему не нужен)

Danik2021 replied:
✔✔✔

Danik2021 committed:
bugfix Ruslan and Denis

section have one name - ✔
style.css in folder css - ✔
script in the end of body - ✔

Danik2021 committed:
homework#2 is ready

add portfolio.html

xrustics commented on index.html:
```diff
@@ -4,109 +4,105 @@
-            </li>
-          </ul>
-        </div>
+      <section class="section-title">
```
не ок для секції ( тег <section) давати класс section-title. Бо по назві - це "заголовок секції" а по факту це блок секції. Якщо ти хочеш дати певний класс для тегу <section> то це може бути більш загальна назва що символізує саме блок, наприклад щось типу:
section, section-block, section-container, section-box, page-section, section-wrapper
Якщо це буде вже якась унікальна секція зі стилями що відрізняються від інших секцій то можна називати по вмісту секції, наприклад:
features-section (секція переваг)

Danik2021 replied:
✔✔✔

xrustics commented on css/style.css:
```diff
@@ -0,0 +1,169 @@
+
+/* Text h2 h3  */
+
+.main-title,
```
Ці класи десь використовуються? Якщо ні видали їх. Бо це виходить зайвий код що буде сповільнити завантаження сторінки сайту. Якщо цей файл вже не використовується - все аналогічно, просто видаляй його

Danik2021 replied:
✔✔✔

Danik2021 committed:
bugFix Ruslan

delete excess style.css - ✔✔✔
section have new names - ✔✔✔

Danik2021 committed:
normalize.css

plug the normalize.css correct

xrustics commented on index.html:
```diff
@@ -4,109 +4,111 @@
-        <h2>Наша команда</h2>
-        <ul class="cont-employee">
+      <section class="section-block">
+        <h2 class="our-team-title">Наша команда</h2>
```
Щось я не знайду в css файлах такого класу... Він використовується? Якщо так - підкажи де він оголошений? Якщо не використовується - просто видали звідси.

Danik2021 replied:
✔✔✔

xrustics commented on index.html:
```diff
@@ -4,109 +4,111 @@
-          </ul>
-        </div>
+      <section class="section-block">
+        <h2 class="what-do-title">Чим ми займаємося</h2>
```
what-do-title - щось цього класу не знаходжу в css файлах. Видали якщо він не використовується

Danik2021 replied:
✔✔✔

xrustics commented:
@Danik2021 переглянь будь ласка всюди класи ( і в css і в  html файлах) познаходь те що не використовується і повидаляй) В проекті не має бути невикористовуємого коду

Danik2021 committed:
bugFix

delete excess class in css - ✔✔✔

Denys320 commented on index.html:
```diff
@@ -71,7 +71,7 @@ <h3>Сучасні Технології</h3>
       <section class="section-block">
-        <h2 class="what-do-title">Чим ми займаємося</h2>
+        <h2>Чим ми займаємося</h2>
         <ul class="ul-img-work list">
```
Я думаю что название класса стало еще сложнее, зачем в теге ul писать в классе то же самое ul, если в теге li есть изображение или еще что то, то это не значит что нужно написать такой сложный класс, "ul-img-work list" можно вообще заменить на "work-list"

Denys320 commented on index.html:
```diff
@@ -71,7 +71,7 @@ <h3>Сучасні Технології</h3>
         </ul>
       </section>
       <section class="section-block">
```
Класс секции можно так же поменять на слово которое наиболее полно характеризует контент, например "work"

Danik2021 committed:
bugFix css

changed hover effects

Danik2021 committed:
!
